### DISCUSSION TITLE
Sonar Cloud - EXECUTION FAILURE ?!

### DISCUSSION THREAD

**Discussion by ilovepixelart** [community/community#53494] [Actions] [answered]
### Select Topic Area

Bug

### Description

Sonar cloud no longer works in GitHub actions since today morning.
Can't find anything about this... in recent threads

node: v16.20.0
npm: 8.19.4
yarn: 1.22.19

Tried switching version of node 16, 18, 20 non of them work, is it GitHub actions issue?
Does anyone experience same thing?

### Body

```bash
INFO: ------------------------------------------------------------------------
INFO: EXECUTION FAILURE
INFO: ------------------------------------------------------------------------
INFO: Total time: 28.779s
ERROR: Error during SonarScanner execution
INFO: Final Memory: 21M/77M
ERROR: 
INFO: ------------------------------------------------------------------------

The version of node.js (12) you have used to run this analysis is deprecated and we stopped accepting it.
Please update to at least node.js 14. You can find more information here: https://docs.sonarcloud.io/appendices/scanner-environment/

ERROR: 
ERROR: Re-run SonarScanner using the -X switch to enable full debug logging.
```

**Comment by ilovepixelart** [comment, marked as answer]
Downgraded 
- from: `sonarsource/sonarcloud-github-action@master`
- to: `sonarsource/sonarcloud-github-action@v1.8`

Looks like its working for me again

**Reply by cequiz** [reply]
Thanks, you saved my day!

**Reply by LBLucaBonetti** [reply]
I have the same problem, where did you get the information to use v1.8?

**Reply by ilovepixelart** [reply]
Was running out of options (used all versions of node)
Then checked official releases they had https://github.com/SonarSource/sonarcloud-github-action/releases
And started experimenting with versions, moving from `master` -> `v1.9` -> `v1.8`

Before that I wrote a message to the support but didn't receive any response yet
Would be better to receive official explanation of this incident

**Comment by desenvolvimento-evolui** [comment]
Very good

**Comment by ilovepixelart** [comment]
If you still watching this, recently upgraded back my pipelines to `sonarsource/sonarcloud-github-action@master` all works fine as before
